scripts/config

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+source _common.sh
+source /usr/share/yunohost/helpers
+
+ynh_abort_if_errors
+
+ynh_app_config_validate() {
+    if [ "$external" -eq "1" ] && [ "$ip_prometheus_server" == "127.0.0.1" ]
+    then
+        ynh_die --message="You need to choose external(s) IP for Promotheus"
+    elif [ "$external" -eq "0" ] && [ "$ip_prometheus_server" != "127.0.0.1" ]
+    then
+        ynh_die --message="You need to choose `127.0.0.1` to listen local Promotheus"
+    fi
+    _ynh_app_config_validate
+}
+
+ynh_app_config_apply() {
+    _ynh_app_config_apply
+
+    ynh_safe_rm "/etc/yunohost/hooks.d/post_iptable_rules/50-nodexport"
+
+    ynh_config_add --template="50-nodexport" --destination="/etc/yunohost/hooks.d/post_iptable_rules/50-nodexport"
+    chmod u+rx "/etc/yunohost/hooks.d/post_iptable_rules/50-nodexport"
```

Comment:
```suggestion
    ynh_config_add --template="50-nodexport" --destination="/etc/yunohost/hooks.d/post_iptable_rules/50-nodexport"
    chmod u+rx "/etc/yunohost/hooks.d/post_iptable_rules/50-nodexport"
```

Shouldn't we wrap with an `if` to only add the hook if Prometheus is not `127.0.0.1` ?

Reply:
Ok, trying refactoring